File: samples/spring-mvc/src/main/java/io/openapiprocessor/samples/NopController.java
package io.openapiprocessor.samples;

import io.openapiprocessor.openapi.api.NopApi;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class NopController implements NopApi {
    private static final Logger log = LoggerFactory.getLogger(NopController.class);

    @Override
    public void getNop() {
        log.info("NopController.getNop()");
    }
}

File: samples/spring-mvc-boot-4-packages-kt/src/main/kotlin/io/openapiprocessor/samples/nop/NopController.kt
/*
 * Copyright 2015 https://github.com/openapi-processor/openapi-processor-samples
 * PDX-License-Identifier: Apache-2.0
 */

package io.openapiprocessor.samples.nop

import org.slf4j.Logger
import org.slf4j.LoggerFactory
import org.springframework.web.bind.annotation.RestController

@RestController
class NopController: NopApi {
    val log: Logger = LoggerFactory.getLogger(this::class.java)

    override fun getNop() {
        log.info("NopController::getNop()")
    }
}
